src/parser/csql_grammar.y

```diff
@@ -3159,6 +3159,32 @@ create_stmt
                                 si->dbname = CONTAINER_AT_2($5);
                                 si->user = CONTAINER_AT_3($5);
                                 si->pwd = CONTAINER_AT_4($5);
+                                if(si->pwd == NULL)
+                                {
+                                   PT_NODE *val = parser_new_node (this_parser, PT_VALUE);
+                                   if (val)                    
+                                     {
+                                        int     err;
+                                        char    cipher[512];
+
+                                        val->type_enum = PT_TYPE_CHAR;
+                                        val->info.value.string_type = ' ';
+
+                                        err = pt_check_dblink_password(this_parser, NULL, cipher, sizeof(cipher));
```

Comment:
It returns error code by `err` value, but the following if-statement only handles an error with `pt_has_error()`.

Reply:
Modified to set the necessary error message inside the pt_check_dblink_password() function.
Thanks for the careful check.